internal/discord/handler/handler.go

```diff
@@ -0,0 +1,43 @@
+package handler
+
+import (
+	"github.com/bwmarrin/discordgo"
+	"github.com/seventv/common/utils"
+	"go.uber.org/zap"
+)
+
+var (
+	DefaultRoleId = ""
```

Comment:
Don't store a variable at the package scope, instead pass the global context and get it from the config.